docs/Testing.md

```diff
@@ -155,11 +155,11 @@ This example is derived from [Testing React Native](https://jestjs.io/docs/tutor
 There are a variety of testing libraries available in React Native that you may find useful to add to your Ignite app
 
 - [React Native Testing Library](https://callstack.github.io/react-native-testing-library/) - port of @testing-library/react for React Native. Useful for unit testing components.
-- [Appium](https://appium.io/) - alternative for integration testing to Detox.
+- [Appium](https://appium.io/) - alternative for integration testing to Maestro.
```

Comment:
We can add Detox here as an alternative

Reply:
I like that idea here, since it used to be our preferred solution